src/features/auth/lib/index.ts

```diff
@@ -0,0 +1,18 @@
+import i18n from 'i18next';
+import * as yup from 'yup';
+
+export const loginSchema = yup.object().shape({
+  email: yup
+    .string()
+    .email(i18n.t('validation.email-format'))
+    .required(i18n.t('validation.email-required')),
```

Comment:
I guess that's why there is no dynamic changes within form validation. i18next.t doesn't switch language in a reactive maner, so you should refresh the page to see those changes